Package.swift

```diff
@@ -506,7 +506,7 @@ let package = Package(
         .package(url: "https://github.com/stencilproject/Stencil", exact: "0.15.1"),
         .package(url: "https://github.com/tuist/GraphViz.git", exact: "0.4.2"),
         .package(url: "https://github.com/SwiftGen/StencilSwiftKit", exact: "2.10.1"),
-        .package(url: "https://github.com/SwiftGen/SwiftGen", exact: "6.6.2"),
+        .package(url: "https://github.com/leszko11/SwiftGen", branch: "feature/xcstrings-support"),
```

Comment:
Code from this branch is an [active PR](https://github.com/tuist/SwiftGen/pull/1) aiming [tuist/SwiftGen](https://github.com/tuist/SwiftGen) repo. I'll update it once it's merged

Reply:
@leszko11 will you end up pushing that upstream to `SwiftGen/SwiftGen`?